pubspec.yaml

```diff
@@ -6,7 +6,7 @@ authors:
 homepage: https://github.com/xqwzts/feedparser
 
 environment:
-  sdk: '>=1.21.0 <2.0.0'
+  sdk: '>=1.21.0 <3.0.0'
```

Comment:
We should also raise the minimum sdk version we're compatible with, as moving to 2.0 is a breaking change, and we can now drop support for older sdks.

sdk: ">=2.0.0-dev.58.0 <3.0.0" should be fine.